### PR TITLE
Add SCRAPE-BDS

### PR DESCRIPTION
**Pull request by arybczak**

### CODE REVIEW

**Comment by arybczak** [comment]
Done.

File: pvss.cabal [hunk 1]
```diff
@@ -27,11 +32,16 @@ library
                      , bytestring
                      , cryptonite
                      , cryptonite-openssl >= 0.3
-                     , foundation
+                     , foundation < 0.0.7
```
Comment (vincenthz) [inline pvss.cabal]: any reason to put an upper bound here ?

**Comment by vincenthz** [comment]
Thanks !

File: pvss.cabal (hunk 1 above)
Comment (arybczak) [inline pvss.cabal]: It doesn't compile with 0.0.7.

**Pull request merged**